derp/derp_server.go

```diff
@@ -1398,3 +1413,14 @@ func writePublicKey(bw *bufio.Writer, key *key.Public) error {
 	}
 	return nil
 }
+
+func (s *Server) ServeDebugTraffic(w http.ResponseWriter, r *http.Request) error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if time.Since(s.lastStatTime) < 5*time.Minute {
```

Comment:
I don't think we'll need this. it should stream regardless, with updates every N seconds (perhaps configurable over a query param)